src/cr/cube/cube.py

```diff
@@ -459,6 +486,32 @@ def _measures(self):
         """
         return _Measures(self._cube_dict, self._all_dimensions)
 
+    @lazyproperty
+    def _numeric_array_dimension(self):
+        """Column dimensions object inflated according to the mean subvariables."""
```

Comment:
dimensions -> dimension

Also, the object itself is not inflated, it's used for inflating other stuff. And we need to sy that it can also return an invalid result if there's no numeric array defined. By the way, should we return a `None` in such case explicitly (like at the start of the method)? I think we should.